### PR TITLE
Newline at PrintToServer end; CamelCase

### LINKED ISSUE #5: camelCase or CamelCase styling for natives
## Description
First natives use camelCase styling, but recently added ones #3 use CamelCase. I think we should decide which one we should go with. Personally I don't mind any of them, but I prefer to keep naming unified.

- Operating system: n/a
- ReHLDS version: n/a
- Metamod version: n/a
- SPMod version: master

## How do we replicate the issue?
n/a

## Expected behavior (i.e. solution)
We should go with camelCase or CamelCase styling for natives, not both.

## Other Comments
What are your thoughts?

### PR DESCRIPTION
**Pull request by voed**
## Description
Moved coreNatives to CamelCase style. Added default newline character at the end of PrintToServer, so 
`1 message == 1 line`

## Motivation and Context
Fix #5 and telegram conversation

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/coreNatives.cpp [hunk 1]
```diff
@@ -17,23 +17,26 @@
...
     char *formatString;
     char bufferOutput[1024];
 
     ctx->LocalToString(params[1], &formatString);
-    gSPGlobal->formatString(bufferOutput, sizeof(bufferOutput), formatString, ctx, params, 2);
-
+    size_t res = gSPGlobal->formatString(bufferOutput, sizeof(bufferOutput), formatString, ctx, params, 2);
+  
+    bufferOutput[res++] = '\n';
+    bufferOutput[res] = '\0';
```
Comment (voed) [inline src/coreNatives.cpp]: not sure its necessary, but its exists in both sourcemod & amxx

**Comment by voed** [comment]
Need to rename filename

File: src/coreNatives.cpp (hunk 1 above)
Comment (Amaroq7) [reply src/coreNatives.cpp]: Just for sanity.

Buffer overflow can occur here. You should subtract ```2``` from buffer size like in the [SourceMod](https://github.com/alliedmodders/sourcemod/blob/master/core/logic/smn_console.cpp#L96) or make a condition like in the [AMX Mod X](https://github.com/alliedmodders/amxmodx/blob/master/amxmodx/amxmodx.cpp#L102).

Comment (voed) [reply src/coreNatives.cpp]: Ah, sure, thats why we need string terminator

**Comment by Amaroq7** [comment]
Thanks. Also for future reference https://help.github.com/articles/closing-issues-using-keywords.

**Pull request merged**